grails-app/controllers/org/pih/warehouse/api/CycleCountApiController.groovy

```diff
@@ -16,17 +17,16 @@ class CycleCountApiController {
 
     CycleCountService cycleCountService
 
-    def getCandidates(CycleCountCandidateFilterCommand filterParams) {
-        List<CycleCountCandidate> candidates = cycleCountService.getCandidates(filterParams, params.facilityId)
+    def getCycleCounts(CycleCountCandidateFilterCommand filterParams) {
```

Comment:
I also don't feel comfortable with both `getCandidates` and `getCycleCounts` - we base those on `CycleCountCandidate`, but we might use the endpoint for different usages - either for get candidates (`cycleCountRequest = null`) or to get cycle counts (to count tab, to resolve tab, to approve tab).

Reply:
yeah this is where the most amount of weirdness happens. However since the response data is structured the same for both pages, I think it's fine to treat this as the original getCandidates (or maybe we can think of a new name that makes the most sense). We should remember that the backend has no knowledge of what the frontend is doing, so the API (and method names) should ideally be reflecting only what the data has (and not what it is being used for). Some other frontend could use the API in a different way.

I'll try to think of a method/API name that makes sense.

Reply:
Good point not to think about it in the frontend way. I always end up making that mistake.
I changed the url to `/cycle-counts/candidates` as suggested and kept the methods with the `candidates` sufix.